### PR TITLE
Fixes #36812 - allow setting (fc)ct_location

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -182,6 +182,10 @@
 #
 # $register_in_foreman::          Register host in Foreman
 #
+# $provisioning_ct_location::     The location of the binary to call when transpiling CoreOS templates.
```
Comment (evgeni) [inline manifests/init.pp:185]: I wasn't sure we really need the `provisioning` prefix here

File: templates/settings.yaml.erb [hunk 2]
```diff
@@ -114,3 +114,10 @@
     :namespace: foreman
 <%   end -%>
 <% end -%>
+
```
Comment (ekohl) [inline templates/settings.yaml.erb:117]: This will result in an empty line at the end and restarts for everyone. Can we avoid this?

Comment (evgeni) [reply templates/settings.yaml.erb:117]: by teaching Puppet to diff YAML by content, and not by checksum? sure ;-)

other than that: there is surely a weird `<%--+-!` invocation to force this, but I can never remember how exactly in ERB.
I can also just move the next line one up

Comment (evgeni) [reply templates/settings.yaml.erb:117]: also, will it really? the line above is `<% end -%>` and I would think the `-%>` should eat up any empty newlines?

**Review by ekohl — APPROVED** [review]

**Pull request merged**